src/Microsoft.Extensions.Configuration.AzureAppConfiguration/DomainVerifier.cs

```diff
@@ -19,7 +20,6 @@ internal static class DomainVerifier
     {
         private const int TlsPort = 443;
         private const string SubjectAltNameOid = "2.5.29.17";
-        private const string DnsName = "DNS Name=";
 
         public static async Task<IEnumerable<string>> GetValidDomains(Uri originEndpoint, string srvHostName)

```

Comment:
I think we need to add a test for this that calls a fake App Config endpoint like azure.aconfig.io and tries to get the domains.

Reply:
Then we expect it fails, get no valid domain?